src/structlog/types.py

```diff
@@ -97,6 +97,30 @@
 """
 
 
+@runtime_checkable
+class ProcessorExceptionFormatter(Protocol):
+    """
+    **Protocol:** A callable that formats an `ExcInfo` into another
+    datastructure.
+
+    The result should be something, that your renderer can work with, e.g., a
```

Comment:
my favorite fellow-German-speaker error that I can't stop making

```suggestion
    The result should be something that your renderer can work with, e.g., a
```

Reply:
🙈